__tests__/app.test.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * @jest-environment node
+ */
+
+const request = require('supertest');
+const nock = require('nock');
+const app = require('../src/app');
+const mockData = require('../src/appMockData');
+
+describe('GET / Homepage', () => {
+  it('should respond with some homepage markup', done => {
+    request(app)
+      .get('/')
+      .then(res => {
+        expect(res.statusCode).toEqual(200);
+        expect(res.text).toContain('Welcome to my jokes API!');
+        done();
+      });
+  });
+});
+
+describe('GET / jokes', () => {
+  it('GET / should serve all jokes!', done => {
```

Comment:
If you are in a describe block that says you are using GET requests for the jokes endpoint, you do not need to state that your test is for a GET request again in your it block.